wordpress-mobile/WordPress-iOS
# Notifications: fix readable media width and adapt separators for readable insets

Turns out, I was wrong in https://github.com/wordpress-mobile/WordPress-iOS/pull/6675#issuecomment-280771279

I simply wasn't following the correct `readableContentGuide` to calculate the `maxMediaEmbedWidth`. Now using the `tableView.readableContentGuide` and `ceil` value.

Also related, however, was that the separator insets used on block cells weren't use the readable insets for drawing. So that also added to the wide image appearing misaligned. I went ahead and fixed this in this same PR for easy review and comparison.

To test:
1. Follow the same steps in https://github.com/wordpress-mobile/WordPress-iOS/pull/6675
2. Also check that the separator above the action buttons is aligned with the left and right readable margins.

Needs review: @frosty can I bug you with another one of these? 😅 No rush.
